vendors/liebearth/drivers/downlink.schema.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This is not a json schema of the downlink, maybe it is an example? Otherwise, the schema is optional, so you can remove this file.